react/details/README.md

```diff
@@ -0,0 +1,92 @@
+# Details Widget
+
+## Widget API
+
+### Attributes
+
+- **locale** (default: `en`)
+- **id**
+
+## i18n
+
+To add a new locale:
+
+- add a new language file under `src/i18n/locales/[newLocaleName].json`
+- edit `src/i18n/locales/index.js` to add new language file
+- language file should start with `translation` as its root object
+
+## API
+
+API helper functions have been added at `src/api/[entity]-api.js`. 4 functions are provided - `get()`, `post()`, `put()`, `delete()`.
```

Comment:
seems that neither the docs nor the code has updated APIs with getById and getAll, as well as camelCase file names